pkg/controller/secret/secret_controller.go

```diff
@@ -595,6 +598,12 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	reqLogger := log.WithValues("Request.Name", request.Name)
 	reqLogger.Info("Reconciling Object")
 
+	if len(os.Getenv("FEDRAMP")) == 0 {
```

Comment:
As opposed to comparing length, there is a LookupEnv function: https://pkg.go.dev/os#LookupEnv

it also allows us to keep the logic in one place, as opposed to us having it here and on L31